server/public/model/integration_action.go

```diff
@@ -356,6 +358,8 @@ type SubmitDialogRequest struct {
 type SubmitDialogResponse struct {
 	Error  string            `json:"error,omitempty"`
 	Errors map[string]string `json:"errors,omitempty"`
+	Type   string            `json:"type,omitempty"`
+	Form   *Dialog           `json:"form,omitempty"`
```

Comment:
It would be nice to have some validation for the new fields. E.g.  `Form` must only be set if Type is `refresh`, right?